src/content/faq/General.md

```diff
@@ -15,45 +15,39 @@ Soufflé candy lollipop [dragée cake chocolate](). Marzipan lemon drops sweet r
 
 ## Is GraphQL a database language like SQL?
 
-<!-- TODO -->
+No, but this is a common misconception. GraphQL is designed for APIs, not databases. It’s a specification for how to request data and it determines how that data is communicated between servers and clients.
+
+GraphQL is frequently used for database interactions though. For example, it can be built into a wrapper around a database or a [GraphQL server](/code/#server-libraries) can query various databases. There are also [services](/code/#services) that implement the GraphQL specification and then allow clients to use GraphQL to query that data source. But it can also be used without a database at all.
 
-Sugar plum gummies donut cheesecake marshmallow donut caramels. Chocolate cake jelly beans ice cream halvah topping lemon drops liquorice croissant. Jelly-o chocolate gummies chocolate I love toffee danish sweet roll. Marzipan I love I love jujubes cupcake bear claw. Soufflé caramels caramels marzipan icing macaroon. Tootsie roll I love jelly beans halvah sweet donut fruitcake. Icing gummi bears tart marshmallow biscuit candy. Candy canes cotton candy soufflé muffin chocolate cake sweet. Candy canes dragée tiramisu gummies danish gummies gingerbread sugar plum gummies.
+For more information on how to get GraphQL to interact with your database, check out our [documentation on resolvers](/learn/execution/#root-fields-resolvers).
 
 ## Does GraphQL replace REST?
 
-<!-- TODO -->
+No, not necessarily. They both handle APIs and [serve the same purpose](/learn/thinking-in-graphs/#business-logic-layer) from a business perspective. GraphQL is considered an alternative to REST, but it’s not a definitive replacement.
 
-Jelly-o cupcake marshmallow marshmallow. Lemon drops cake caramels donut halvah cupcake topping tootsie roll. Jelly cotton candy candy cheesecake danish [jelly-o icing bonbon]().
+GraphQL and REST can actually co-exist in your stack. For example, you can abstract REST APIs behind a [GraphQL server](/code/#server-libraries) or mask your REST endpoint into a GraphQL endpoint using [resolvers](/learn/execution/#root-fields-resolvers). For an opinionated perspective on how GraphQL compares to REST, check out [How To GraphQL](https://www.howtographql.com/basics/1-graphql-is-the-better-rest/).
```

Comment:
> you can abstract REST APIs behind a [GraphQL server](/code/#server-libraries)

> mask your REST endpoint into a GraphQL endpoint using [resolvers](/learn/execution/#root-fields-resolvers)

I think the second statement is one of the approaches to achieve the first one.

```suggestion
GraphQL and REST can actually co-exist in your stack. For example, you can abstract REST APIs behind a [GraphQL server](/code/#server-libraries), building proof of concept can be as simple as masking your REST endpoint into a GraphQL endpoint using [root resolvers](/learn/execution/#root-fields-resolvers). For an opinionated perspective on how GraphQL compares to REST, check out [How To GraphQL](https://www.howtographql.com/basics/1-graphql-is-the-better-rest/).
```

Reply:
Pushed a different version. A bit different wording than what you suggested, but same concept. Let me know if you want me to update it more!